docker-compose.dev.yml

```diff
@@ -38,7 +38,7 @@ services:
 #      - motoserver
   dev_database:
     container_name: pixetl-dev-database
-    image: postgis/postgis:11-3.0-alpine
+    image: postgis/postgis:12-3.3-alpine
```

Comment:
Github won't let me leave a comment on line 57 below, but does it make sense to update line 57 to 50000:50000 for the moto port, even though it is commented out, to make sure it is consistent if uncommented.

Or should we just remove lines 53-59 below to remove confusion?

Reply:
Good point, will change!